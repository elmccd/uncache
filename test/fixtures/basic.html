text1
<!-- uncache(rename: false, append:123) --><link rel="stylesheet" href="src/css/index.css"/><!-- enduncache -->
text2
<!-- uncache(rename: true, append:123) -->
<script src="src/js/file1.js"></script>
<script src="src/js/file2.js"></script>
<!-- enduncache -->
text3
<!-- uncache(rename: true, append:hash) -->
<script src="file1.js"></script>
<!-- enduncache -->
<!-- uncache(rename: false, append:hash) -->
<script src="src/js/file1.js"></script>
<!-- enduncache -->
<!-- uncache(rename: true, template:{{path}}{{name}}-{{append}}-cached.{{extension}}) -->
<script src="src/js/file1.js"></script>
<!-- enduncache -->
<!-- uncache(rename: true, append:123) -->
<link rel="stylesheet" href="not-existing-file.js"/>
<!-- enduncache -->