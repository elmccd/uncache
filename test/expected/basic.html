text1
<link rel="stylesheet" href="src/css/index.css?123"/>
text2
<script src="src/js/file1_123.js"></script>
<script src="src/js/file2_123.js"></script>
text3
<script src="file1_557f44c66d.js"></script>
<script src="src/js/file1.js?557f44c66d"></script>
<script src="src/js/file1-557f44c66d-cached.js"></script>
<link rel="stylesheet" href="not-existing-file.js"/>